### PR TITLE
F* Added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#401 
#### What does this implement/fix? Explain your changes.
By meting this PR bellow changes will take place:
- `F*` class statistics

As we found out the _Jaccard Index_ and the _F*_ are the same for our scope so we don't need to redefine this statistic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -922,6 +922,8 @@ PyCM can be used online in interactive Jupyter Notebooks via the Binder or Colab
 
 <blockquote>76- N. A. Macmillan and C. D. Creelman, <i>Detectiontheory: A user's guide</i>. Psychology press, 2004.</blockquote>
 
+<blockquote>77- Hand, David J., Peter Christen, and Nishadi Kirielle. "F*: an interpretable transformation of the F-measure." <i>Machine Learning</i> 110.3 (2021): 451-456.</blockquote>
```
Comment (sepandhaghighi) [inline README.md:925]: @alirezazolanvari Would you please review this PR?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `F*` class statistic
```
Comment (alirezazolanvari) [inline CHANGELOG.md:9]: I'm not sure if this is correct to say F* is added. What's your opinion? 
My suggestion is to mention in the Changed section as an update for the document of Jaccard index.

Comment (sepandhaghighi) [reply CHANGELOG.md:9]: Yeah, you are right!
@sadrasabouri

File: README.md (hunk 1 above)
Comment (sepandhaghighi) [reply README.md:925]: We need `IEEE` format.
@alirezazolanvari @sadrasabouri

Comment (alirezazolanvari) [reply README.md:925]: Here is the reference with the correct format.

D. J. Hand, P. Christen, and N. Kirielle, "F*: an interpretable transformation of the F-measure," <i>Machine Learning</i>, vol. 110, no. 3, pp. 451-456, 2021.

**Review by alirezazolanvari — APPROVED** [review]
Good job!🍻

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯

**Pull request merged**